### PR TITLE
CNX-9434 use proposed naming standard for all DUI3 converter classes

### PR DESCRIPTION
**Pull request by adamhathcock**
CNX-9434

### CODE REVIEW

File: DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RevitRootObjectBuilder.cs [hunk 1]
```diff
@@ -8,24 +8,17 @@
 
 namespace Speckle.Connectors.Revit.Operations.Send;
 
-public class RootObjectBuilder : IRootObjectBuilder<ElementId>
+public class RevitRootObjectBuilder : IRootObjectBuilder<ElementId>
 {
   // POC: SendSelection and RevitConversionContextStack should be interfaces, former needs interfaces
-  private readonly ISpeckleConverterToSpeckle _converter;
-  private readonly ToSpeckleConvertedObjectsCache _convertedObjectsCache;
```
Comment (AlanRynne) [inline DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RevitRootObjectBuilder.cs:15]: Was the object cache not being used at all?

File: DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3.DependencyInjection/ArcGISConverterModule.cs [hunk 2]
```diff
@@ -11,9 +11,9 @@ public class ArcGISConverterModule : ISpeckleModule
 {
   public void Load(SpeckleContainerBuilder builder)
   {
-    builder.AddConverterCommon();
```
Comment (AlanRynne) [inline DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3.DependencyInjection/ArcGISConverterModule.cs:14]: 🤔 Don't we need this call anymore?

File: DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/FakeTopLevelConverter.cs [hunk 3]
```diff
@@ -6,11 +6,11 @@
 namespace Speckle.Converters.ArcGIS3;
 
 [NameAndRankValue(nameof(String), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
-public class FakeConversion : IHostObjectToSpeckleConversion, IRawConversion<String, Point>
+public class FakeTopLevelConverter : IToSpeckleTopLevelConverter, ITypedConverter<String, Point>
```
Comment (AlanRynne) [inline DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/FakeTopLevelConverter.cs:9]: This kinda gives me a headache 🤕 Not your doing, just flagging this to myself to open a ticket about it.

**Review by AlanRynne — COMMENTED** [review]
All good for now, just half way through... more to come!

**Review by AlanRynne — CHANGES_REQUESTED** [review]
"Comments don't count as reviews"... Flagging as changes requested, just to get the "since your last review" feature to activate.

File: DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RevitRootObjectBuilder.cs (hunk 1 above)
Comment (adamhathcock) [reply DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RevitRootObjectBuilder.cs:15]: It wasn't!

**Review by AlanRynne — APPROVED** [review]

File: DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3.DependencyInjection/ArcGISConverterModule.cs (hunk 2 above)
Comment (adamhathcock) [reply DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3.DependencyInjection/ArcGISConverterModule.cs:14]: it's a slightly different call now

**Pull request merged**